test/controllers/posts_controller_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test-helper'
```

Comment:
- [ ] Following the project requirements, you should use RSpec gem for testing. Please add RSpec as a gem to your project and set up tests on the controllers using RSpec.